site/roles/trinity/repos/tasks/main.yml

```diff
@@ -35,7 +35,7 @@
 
 - name: Import GPG keys
   rpm_key:
-    key: "{{ item.key }}"
+    key: "{{ item.key | regex_replace('^file://') }}"
```

Comment:
Requiring prepending a local path to a key with `file://` creates a special case, which we'd like to avoid for the sake of consistency. It seems, we can drop the `gpgkey: "{{ item.key|default([]) }}"` line completely from the task below, because at that point we'll already have the GPG key imported.